test/core/shared/sdk/embeddedSdk.test.js

```diff
@@ -65,5 +66,20 @@ describe('EmbeddedSDK', () => {
       should(embeddedSdk.protocol.query)
         .be.calledWithMatch({ ...request, propagate: true });
     });
+
+    it('should throw an error if the action is forbidden', async () => {
+      const forbiddenRequest = { controller: 'auth', action: 'createApiKey' };
+
+      should(() => embeddedSdk.query(forbiddenRequest)).throw(Error);
```

Comment:
Usually we try to check the error code (at least) and error class since changing them is a breaking change